webpush/encrypt_test.go

```diff
@@ -202,6 +231,39 @@ func TestRfcVectors(t *testing.T) {
 	}
 }
 
+// TestAes128gcmRfcVectors uses the values given in the RFC for HTTP encryption to verify
+// that the code conforms to the RFC
+// See: https://tools.ietf.org/html/draft-ietf-webpush-encryption-07#appendix-A
+func TestAes128gcmRfcVectors(t *testing.T) {
```

Comment:
It actually makes it harder to read the function code with so many empty lines between expressions.